src/Framework/AspNetCoreAnalyzers/test/Http/HeaderDictionaryAddAnalyzerTests.cs

```diff
@@ -0,0 +1,65 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.CodeAnalysis.Testing;
+using VerifyCS = Microsoft.AspNetCore.Analyzers.Verifiers.CSharpAnalyzerVerifier<
+    Microsoft.AspNetCore.Analyzers.Http.HeaderDictionaryAddAnalyzer>;
+
+namespace Microsoft.AspNetCore.Analyzers.Http;
+
+public class HeaderDictionaryAddAnalyzerTests
+{
+    [Fact]
+    public async Task IHeaderDictionary_WithAdd_ReportsDiagnostics()
```

Comment:
This test is redundant. It's already tested by the codefix tests.
It's also a better practice to have a single test file for both analyzers codefixes.

Reply:
Ah, that's a good point. I've removed this test and merged to these two remaining analyzer tests and the code fix tests into a single test file.